htsohm/db/material.py

```diff
@@ -127,6 +129,8 @@ class Material(Base):
     surface_area_bin = Column(Integer)                        # dimm.
     void_fraction_bin = Column(Integer)                       # dimm.
 
+    # relationships
+    structure = relationship("Structure", uselist=False, back_populates="materials")
```

Comment:
is back_populates necessary here? Have you read all of this: http://docs.sqlalchemy.org/en/latest/orm/backref.html#relationships-backref